src/greenlight_alby_client.rs

```diff
@@ -775,6 +775,45 @@ impl From<cln::WithdrawResponse> for WithdrawResponse {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct CloseRequest {
+    pub id: String,
+    pub unilateral_timeout: Option<u32>,
+    pub destination: Option<String>,
+    pub fee_negotiation_step: Option<String>,
+    pub force_lease_closed: Option<bool>,
+}
+
+impl From<CloseRequest> for cln::CloseRequest {
+    fn from(req: CloseRequest) -> Self {
+        cln::CloseRequest {
+            id: req.id,
+            unilateraltimeout: req.unilateral_timeout,
```

Comment:
```suggestion
            unilateral_timeout: req.unilateral_timeout,
```

Reply:
I agree it's kinda ugly but this field's name comes from Greenlight — we cannot change it

Reply:
ah, thanks!